pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+        vArg2 = ["NAME:PathSweepParameters"]
+        vArg2.append("DraftAngle:="), vArg2.append(self._arg_with_dim(draft_angle, "deg"))
+        vArg2.append("DraftType:="), vArg2.append(GeometryOperators.draft_type_str(draft_type))
+        vArg2.append("CheckFaceFaceIntersection:="), vArg2.append(is_check_face_intersection)
+        vArg2.append("TwistAngle:="), vArg2.append(str(twist_angle) + "deg")
+
+        self.oeditor.SweepAlongPath(vArg1, vArg2)
+
+        return self.update_object(objid)
+
+    @pyaedt_function_handler()
+    def sweep_around_axis(self, objid, cs_axis, sweep_angle=360, draft_angle=0, number_of_segments=0):
+        """Sweep the selection around the axis.
+
+        Parameters
+        ----------
+        objid : list, str, int, :class:`pyaedt.modeler.Object3d.Object3d`
+            Name or ID of the object.
+        cs_axis :
+            Coordinate system axis or the Application.AXIS object.
+        sweep_angle : float
+            Sweep angle in degrees. The default is ``360``.
+        draft_angle : float
+            Draft angle in degrees. The default is ``0``.
+        number_of_segments : int, optional
+            Number of segments of the sweep operation. Default is ``0``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.SweepAroundAxis
+        """
+        selections = self.convert_to_selections(objid)
+
+        vArg1 = ["NAME:Selections", "Selections:=", selections, "NewPartsModelFlag:=", "Model"]
+        vArg2 = [
+            "NAME:AxisSweepParameters",
+            "DraftAngle:=",
+            self._arg_with_dim(draft_angle, "deg"),
+            "DraftType:=",
+            "Round",
+            "CheckFaceFaceIntersection:=",
+            False,
+            "SweepAxis:=",
+            GeometryOperators.cs_axis_str(cs_axis),
+            "SweepAngle:=",
+            self._arg_with_dim(sweep_angle, "deg"),
+            "NumOfSegments:=",
+            str(number_of_segments),
+        ]
+
+        self.oeditor.SweepAroundAxis(vArg1, vArg2)
+
+        return self.update_object(objid)
+
+    @pyaedt_function_handler()
+    def section(self, object_list, plane, create_new=True, section_cross_object=False):
+        """Section the selection.
+
+        Parameters
+        ----------
+        object_list : list, str, int, or  :class:`pyaedt.modeler.Object3d.Object3d`
+            One or more objects to section.
+        plane : str
+            Coordinate plane or Application.PLANE object.
+            Choices for the coordinate plane are ``"XY"``, ``"YZ"``, and ``"ZX"``.'
+        create_new : bool, optional
+            The default is ``True``, but this parameter has no effect.
+        section_cross_object : bool, optional
+            The default is ``False``, but this parameter has no effect.
```

Comment:
Not sure what these parameters are here if they have no affect. They should still have descriptions though, shouldn't they?